extension/app/scripts.babel/pwa-check.js

```diff
@@ -105,22 +120,33 @@ function injectIntoTab(chrome, fnPair) {
   });
 }
 
-function executeInTab(chrome, fn) {
-  return new Promise((res, reject) => {
-    chrome.tabs.executeScript(null, {
-      code: `(${fn.toString()})()`
-    }, ret => {
-      res(ret);
-    });
+function runAudits(chrome, audits) {
+  const fnString = audits.reduce((prevValue, audit, index) => {
+    return prevValue + (index > 0 ? ',' : '') +
+        `new Promise(function(resolve, reject) {
```

Comment:
can we pull lines 126 -> 131 into a named fn that returns this wrapped string fn?